iot_verifier/src/poc.rs

```diff
@@ -743,6 +782,20 @@ mod tests {
         assert!(100 > (gmaps_dist.abs_diff(dist) as i32));
     }
 
+    #[test]
+    fn test_calc_cell_distance() {
```

Comment:
We probably want at least one negative test (two cells that are less than the min).

Reply:
The negative test will be part of the test covering the verification itself, as per the other verifications.  Will add shortly

Reply:
verification test added here: 0c266f8131af8ba7ca0a1b955603a166f8215314